src/engraving/libmscore/engravingitem.h

```diff
@@ -213,29 +213,6 @@ class EngravingItem : public EngravingObject
     OffsetChange offsetChanged() const { return m_offsetChanged; }
     void setOffsetChanged(bool v, bool absolute = true, const PointF& diff = PointF());
 
-    inline void doSetPos(double x, double y)
-    {
-        m_pos.setX(x),
-        m_pos.setY(y);
-    }
-
-    const PointF& ipos() const { return m_pos; }
-    virtual const PointF pos() const { return m_pos + m_offset; }
-    virtual double x() const { return m_pos.x() + m_offset.x(); }
-    virtual double y() const { return m_pos.y() + m_offset.y(); }
-    virtual void setPos(double x, double y) { doSetPos(x, y); }
-    virtual void setPos(const PointF& p) { doSetPos(p.x(), p.y()); }
-    void setPosX(double x) { doSetPos(x, m_pos.y()); }
-    void setPosY(double y) { doSetPos(m_pos.x(), y); }
-    void movePos(const PointF& p) { doSetPos(m_pos.x() + p.x(), m_pos.y() + p.y()); }
-    void movePosX(double x) { doSetPos(m_pos.x() + x, m_pos.y()); }
-    void movePosY(double y) { doSetPos(m_pos.x(), m_pos.y() + y); }
-    double xpos() { return m_pos.x(); }
-    double ypos() { return m_pos.y(); }
-    virtual void move(const PointF& s) { m_pos += s; }
-    bool skipDraw() const { return m_skipDraw; }
-    void setSkipDraw(bool val) { m_skipDraw = val; }
-
     virtual PointF pagePos() const;            ///< position in page coordinates
     virtual PointF canvasPos() const;          ///< position in canvas coordinates
     double pageX() const;
```

Comment:
These methods here, and many below, actually depend on the LayoutData, and on the methods that you've moved below the definition of the LayoutData struct. Would it be an idea to move these methods below those LayoutData-related methods?

Reply:
I completely agree, I have not yet put the entire interface in order, only the most obvious thing that I have now faced.
(I hope we can safely remove some of these methods, it seems that their number is excessive)